llvm/lib/Target/Mips/MCTargetDesc/MipsMCAsmInfo.cpp

```diff
@@ -51,3 +51,25 @@ MipsELFMCAsmInfo::MipsELFMCAsmInfo(const Triple &TheTriple,
   DwarfRegNumForCFI = true;
   HasMipsExpressions = true;
 }
+
+void MipsCOFFMCAsmInfoMicrosoft::anchor() { }
+
+MipsCOFFMCAsmInfoMicrosoft::MipsCOFFMCAsmInfoMicrosoft() {
+  WinEHEncodingType = WinEH::EncodingType::Itanium;
+  CommentString = ";";
```

Comment:
In general, I would recommend against using `;` as comment string here.

While this is the asm dialect for Microsoft variants, this still is within the scope of GNU assembler style, produced and consumed by clang and `llvm-mc` and similar tools.

For the full standalone Microsoft style assembly, on x86 we have the separate `llvm-ml` tool. And on aarch64, we've changed `CommentString` to use `//` for the microsoft variant, just like for the GNU/ELF variants, in order to be able to build e.g. compiler-rt assembly sources for the microsoft variants too - see 71c29b4cf3fb2b5610991bfbc12b8bda97d60005.